amps/ags/rm-community/rm-community-repo/pom.xml

```diff
@@ -15,6 +15,12 @@
       <app.amp.client.war.folder>${project.build.directory}/${project.build.finalName}-war</app.amp.client.war.folder>
 
       <image.name>alfresco/alfresco-governance-repository-community-base</image.name>
+
+      <local.registry.host>127.0.0.1</local.registry.host>
+      <local.registry.port>5000</local.registry.port>
```

Comment:
Should these be called `local.xxx` or is there any scenario where we might want to override these values from e.g.: `127.0.0.1` to a remote registry/port combination, so that we can for example build a version of this AGS Docker image starting from a remotely available tagged one?

Ideally, if this scenario is a valid scenario and works as expected, these properties should rather be called just `registry.host` and `registry.port`.

Reply:
Done